table/block_based/partitioned_index_reader.cc

```diff
@@ -140,13 +142,15 @@ Status PartitionIndexReader::CacheDependencies(
   index_block.GetValue()->NewIndexIterator(
       internal_comparator()->user_comparator(),
       rep->get_global_seqno(BlockType::kIndex), &biter, kNullStats, true,
-      index_has_first_key(), index_key_includes_seq(), index_value_is_full());
+      index_has_first_key(), index_key_includes_seq(), index_value_is_full(),
+      false /* block_contents_pinned */, user_defined_timestamps_persisted());
   // Index partitions are assumed to be consecuitive. Prefetch them all.
   // Read the first block offset
   biter.SeekToFirst();
   if (!biter.Valid()) {
     // Empty index.
     return biter.status();
+    return biter.status();
```

Comment:
Minor: some accidental code duplication going on here ;)

Reply:
oops, thank you for catching this!